uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -355,6 +391,23 @@ =head2 Methods
             . "inserted into the database \n";
 spool($message,'N', $notify_notsummary);
 
+################################################################
+############### Email last completion message ##################
+################################################################
+my @inserted_files = $imaging_upload->getInsertedFileNamesUsingUploadID($upload_id);
+
+$message = "\n$minc_created minc file(s) created, "
+            . "and $minc_inserted minc file(s) "
+            . "inserted into the database: \n";
+    my $mail_subject = "IMAGING_UPLOAD_FILE: $uploaded_file insertion completed.";
+    my $mail_message = "Progress: Success\n".$message."\n"
+                      . join("\n", @inserted_files)."\n";
```

Comment:
Just tested it and this is what the $mail_message contains:
```
0  'Progress: Success

13 minc file(s) created, and 13 minc file(s) inserted into the database: 

ARRAY(0x7fe6d3a46d40)
'
```

The problem is the `@inserted_files` has the following structure:
```
0  ARRAY(0x7ff2cfebe900)
   0  ARRAY(0x7ff2cfebe828)
      0  'assembly/400248/V6/mri/native/demo_400248_V6_t1_005.mnc'
   1  ARRAY(0x7ff2cfeec048)
      0  'assembly/400248/V6/mri/native/demo_400248_V6_t2_005.mnc'
   2  ARRAY(0x7ff2cfeebdd8)
      0  'assembly/400248/V6/mri/native/demo_400248_V6_fieldmapDWI_004.mnc'
   3  ARRAY(0x7ff2cfe59530)
      0  'assembly/400248/V6/mri/native/demo_400248_V6_fieldmapBOLD_013.mnc'
   4  ARRAY(0x7ff2cfeebda8)
```

Related to your comment in https://github.com/aces/Loris-MRI/pull/578/files#r578689679

Reply:
this should be resolved by [this](https://github.com/aces/Loris-MRI/pull/578/commits/2c6da611c68b2f09b2899ed8ac8f38a15039214c)